l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -196,10 +199,12 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
         return IStateTransitionManager(stateTransitionManager[_chainId]).getHyperchain(_chainId);
     }
 
+    /// @notice return the stm asset id of a chain.
```

Comment:
params (I will not leave comments about documenting parameters/returned values in the subsequent functions, please go through all of them)